File: 4.c
#include<stdio.h>
int main()
{
struct demo
{
  int arr[3];
  float brr[2];
};
 struct demo obj;
 obj.arr[0]=11;
 obj.arr[1]=33;
 obj.arr[2]=44;
 obj.brr[0]=12.44;
 obj.brr[1]=10.11;

printf("%d\n%d\n%d\n%f\n%f\n", obj.arr[0], obj.arr[1],obj.arr[2],obj.brr[0],obj.brr[1]c );
return 0;
}

File: 4.java
import java.lang.*;
import java.util.*;

class program4
{
 public static void main(String arg[])
{
int ret=0;

 number obj=new number(5);
 obj.accept();
 obj.display();
ret=obj.summation();
System.out.println("summation is:"+ret);

}
}

class number
{
 private int arr[];

public number(int no)
{
 arr=new int[no];
}

public void accept()
{
int icnt=0;

Scanner sobj=new Scanner(System.in);
System.out.println("enter the number of array:");
for(icnt=0;icnt<arr.length;icnt++)
{
   arr[icnt]=sobj.nextInt();
}
}

public  void display()
{
int icnt=0;
System.out.println("enter the number of array element:");
for(icnt=0;icnt<arr.length;icnt++)
{
 System.out.println(arr[icnt]);
}
}

int summation()
{
 int icnt=0;
int sum=0;

for(icnt=0;icnt<arr.length;icnt++)
{
  sum=sum+icnt;
}
return sum;
}
}


